defs/go.bzl

```diff
@@ -1,3 +1,6 @@
+load("@io_bazel_rules_go//go/private:common.bzl", "get_go_toolchain")
+load("@io_bazel_rules_go//go/private:providers.bzl", "GoSource")
```

Comment:
You can load this from @io_bazel_rules_go//go:def.bzl. The fields aren't documented yet, but it is exported already.